.appveyor.yml

```diff
@@ -0,0 +1,23 @@
+# Operating system (build VM template)
+os: Visual Studio 2015
+
+# clone directory
+clone_folder: c:\xmr-stak-cpu
+
+install:
+  - call "%VS140COMNTOOLS%..\..\VC\vcvarsall.bat" x64
```

Comment:
IMO this could be the reason for your build issues, you need to load VsMSBuildCmd.bat to get the environment configuard to allow compiling.